common/transport/amqp/src/amqp.ts

```diff
@@ -3,23 +3,20 @@
 
 'use strict';
 
-import * as amqp10 from 'amqp10';
 import * as machina from 'machina';
+import urlParser = require('url');
```

Comment:
>require('url') [](start = 19, length = 14)

 why use require instead of import from?

Reply:
I vaguely remember adding this.  I think I had some difficulty doing from?  I'll go back and redo and see what happens.

---
In reply to: [209127671](https://github.com/Azure/azure-iot-sdk-node/pull/344#discussion_r209127671) [](ancestors = 209127671)